src/main/java/org/openRealmOfStars/AI/PlanetHandling/PlanetHandling.java

```diff
@@ -550,6 +550,9 @@ private static void handlePlanetPopulation(final Planet planet,
           planet.setWorkers(Planet.RESEARCH_SCIENTIST, 2);
           break;
         }
+        default: {
```

Comment:
Just marking for myself that this looks like a bug. It does not make sense select workers according total workers in a switch case.